service_configuration_lib/spark_config.py

```diff
@@ -334,7 +359,24 @@ def get_dra_configs(spark_opts: Dict[str, str]) -> Dict[str, str]:
         log.warning(
             f'\nSetting spark.dynamicAllocation.maxExecutors as {max_executors}. If you wish to '
             f'change the value of maximum executors, please provide the exact value of '
-            f'spark.dynamicAllocation.maxExecutors in --spark-args\n',
+            f'spark.dynamicAllocation.maxExecutors in your spark args\n',
+        )
+
+    # TODO: add regex to better match Jupyterhub Spark session app name
+    if 'jupyterhub' in spark_app_name and 'spark.dynamicAllocation.initialExecutors' not in spark_opts:
+        if min_ratio_executors:
```

Comment:
This is confusing.
`min_ratio_executors=min_executors` can also be `0`.
The better check and initialization probably is:
`min_executors = None`
if min_executors is not None:

Reply:
Okay. I can do that. Was just trying to keep the type same